File: Visual C#/SRC/Code/Option/Form1.cs
namespace Option
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            radioButton1.Checked = true;
        }

        private void button1_Click(object sender, EventArgs e)
        {
            listBox1.Items.Clear();

            if (radioButton1.Checked)
                listBox1.Items.Add(radioButton1.Text);
            if (radioButton2.Checked)
                listBox1.Items.Add(radioButton2.Text);
            if (checkBox1.Checked)
                listBox1.Items.Add(checkBox1.Text);
            if (checkBox2.Checked)
                listBox1.Items.Add(checkBox2.Text);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            listBox1.Items.Clear();
            radioButton1.Checked = true;
            checkBox1.Checked = false;
            checkBox2.Checked = false;
        }
    }


}

File: Visual Basic/PROJECTS/repos/PizzaPicker/PizzaPicker/Form1.vb
Public Class Form1
    Private Sub BtnShow_Click(sender As Object, e As EventArgs) Handles BtnShow.Click
        LstPizza.Items.Clear()
        If RadioButton1.Checked = True Then
            LstPizza.Items.Add(RadioButton1.Text)
        End If
        If RadioButton2.Checked = True Then
            LstPizza.Items.Add(RadioButton2.Text)
        End If

        If CheckBox1.Checked = True Then
            LstPizza.Items.Add(CheckBox1.Text)
        End If
        If CheckBox2.Checked = True Then
            LstPizza.Items.Add(CheckBox2.Text)
        End If

    End Sub

    Private Sub BtnClear_Click(sender As Object, e As EventArgs) Handles BtnClear.Click
        LstPizza.Items.Clear()
        RadioButton1.Checked = False
        RadioButton2.Checked = False
        CheckBox1.Checked = False
        CheckBox2.Checked = False
    End Sub
End Class
